src/arch/ofi/machine.C

```diff
@@ -1104,19 +1792,28 @@ void process_long_send_ack(struct fi_cq_tagged_entry *e, OFIRequest *req)
      * An OFIRmaAck was received; Close memory region and free original msg.
```

Comment:
The memory region is no longer closed (commented out in code below). Either code or comment is wrong.

Reply:
Comment was out of date, the close is not connect there.  The free will either return it to the pool, or truly free and close if the buffer is out of pool.